bottlecap/src/traces/http_utils.rs

```diff
@@ -0,0 +1,162 @@
+// Copyright 2023-Present Datadog, Inc. https://www.datadoghq.com/
+// SPDX-License-Identifier: Apache-2.0
+
+use hyper::{
+    header,
+    http::{self, HeaderMap},
+    Body, Response, StatusCode,
+};
+use log::{error, info};
+use serde_json::json;
+
+/// Does two things:
+/// 1. Logs the given message. A success status code (within 200-299) will cause an info log to be
+///    written,
+/// otherwise error will be written.
+/// 2. Returns the given message in the body of JSON response with the given status code.
+///
+/// Response body format:
+/// {
+///     "message": message
+/// }
+pub fn log_and_create_http_response(
+    message: &str,
+    status: StatusCode,
+) -> http::Result<Response<Body>> {
+    if status.is_success() {
+        info!("{message}");
```

Comment:
```suggestion
        debug!("{message}");
```

Maybe we don't want to info this?, for the error I think that's fine, but not for the success, WDYT?